app/Resources/views/app/main/estimate.html.twig

```diff
@@ -0,0 +1,91 @@
+                                <input class="form-check-input" type="radio" name="gridRadios" id="gridRadios2">
+                                {{ 'estimate_status2' |trans |upper }}
+                            </label>
+                        </div>
+                    </div>
+                    <div class="col-md-offset-3 col-md-6 form-group">
+                        <label class="estimateForm-center--font" for="usr">{{ 'estimate_fullName' |trans |upper }}</label>
+                        <input type="text" class="form-control" id="usr">
+                    </div>
+                    <div class="col-md-offset-3 col-md-6" id="estimateRestaurant">
+                        <label class="estimateForm-center--font" for="rest">{{ 'estimate_restaurant' |trans |upper }}</label>
+                    </div>
+                    <div class="col-md-offset-3 col-md-6 estimateForm-center--labelOut">
+                        <div class="input-group">
+                            <div class="input-group-btn">
+                                <button type="button" class="btn btn-secondary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                    {{ 'estimate_restaurantChoice' |trans |upper }}
+                                </button>
+                                <ul class="dropdown-menu" role="menu">
+                                    <li><a class="dropdown-item" href="#">Wild Restaurant</a></li>
+                                    <li><a class="dropdown-item" href="#">Un Monde Vegan</a></li>
+                                    <li><a class="dropdown-item" href="#">Un Monde Carnivore</a></li>
+                                </ul>
+                            </div>
+                            <input type="text" class="form-control" aria-label="Text input with dropdown button">
+                        </div>
+                    </div>
+                    <div class="col-md-offset-3 col-md-6 estimateForm-center--labelOut">
+                        <label class="estimateForm-center--font" id="estimateDate" for="date">{{ 'estimate_date' |trans |upper }}</label>
+                    </div>
+                    <div class='col-md-offset-3 col-md-6'>
+                        <div class="form-group">
+                            <div class='input-group date datepicker' id='datetimepicker1'>
+                                <input type='text' class="form-control datepicker" />
+                                <span class="input-group-addon">
+                                    <span class="glyphicon glyphicon-calendar"></span>
+                                </span>
+                            </div>
+                        </div>
+                    </div>
+                    <div class=" col-md-offset-3 col-md-6 form-group">
+                        <label class="estimateForm-center--font" for="people">{{ 'estimate_numberPeople' |trans |upper }}</label>
```

Comment:
In my view I can add -1 etc.. people. Maybe find a way to block the counter to 0